static/app/utils/replays/hydrateFrames.tsx

```diff
@@ -25,10 +25,6 @@ export default function hydrateFrames(attachments: unknown[]) {
       return;
     }
     if (isBreadcrumbFrameEvent(attachment)) {
-      if (attachment.data.payload.category === 'sentry.feedback') {
-        // @ts-expect-error In SDK <= 7.100.0 we were incorrectly setting the timestamp
-        attachment.data.payload.timestamp = attachment.data.timestamp;
-      }
       breadcrumbFrames.push(attachment.data.payload);
```

Comment:
Lets tweak this if-statement so we just drop these categories on the floor instead of adding them to the list. If we ever had a list that was showing like 'all other breadcrumb types' then a check here will mean that `sentry.feedback` is removed from that type of list.

```suggestion
      if (attachment.data.payload.category !== 'sentry.feedback') {
        breadcrumbFrames.push(attachment.data.payload);
      }
```